nginx/conf.d/redirects/redirects.conf

```diff
@@ -4667,3 +4667,7 @@ rewrite ^/listings/va-caribbean-healthcare-system-2114025680/?$ https://$host/lo
 rewrite ^/top-rated-treatment-centers/?$ https://$host/top-rated-rehabs/ permanent;
 #REHCORE-16
 rewrite ^/not-ready-to-talk/?$ https://$host/text-support-sign-up/ permanent;
+
+#DIR-2406
+rewrite ^/treatment/heroin/?$ https://$host/heroin/treatment/ permanent;
+rewrite ^/treatment/fentanyl/?$ https://$host/fentanyl/treatment/ permanent;
```

Comment:
based on the task looks like /treatment/fentanyl/ is being redirected to /opioids/fentanyl/treatment/ instead

Reply:
hey @misunchoi  is there no staging branch here for QA?

Reply:
we can QA this on staging once this is merged to the main branch, we'll just need to redeploy the frontend app on staging to see the updates